### PR TITLE
Upgrade NwBuilder to version 3.7.4

### PR DESCRIPTION
**Pull request by McGiverGim**
Upgrade to latest version.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — DISMISSED** [review]

**Comment by McGiverGim** [comment]
Upgraded to version 3.7.4... let's see if we can merge this soon :)

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
